<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="Coerce an EML file or object into an emld object."><title>Coerce an EML file or object into an emld object. — as_emld • emld</title><link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="/apple-touch-icon.png"><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://docs.ropensci.org/bootstrap-toc.min.js"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Coerce an EML file or object into an emld object. — as_emld"><meta property="og:description" content="Coerce an EML file or object into an emld object."><meta property="og:image" content="https://docs.ropensci.org/emld/logo.png"><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Matomo --><link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/cookieconsent@3/build/cookieconsent.min.css"><script src="https://cdn.jsdelivr.net/npm/cookieconsent@3/build/cookieconsent.min.js" data-cfasync="false"></script><script src="https://ropensci.org/scripts/matomo.js"></script><noscript><p><img src="https://ropensci.matomo.cloud/matomo.php?idsite=1&amp;rec=1" style="border:0;" alt=""></p></noscript><!-- End Matomo Code --></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    <a href="https://ropensci.org" class="external-link"><img src="https://ropensci.org/img/icon_short_white.svg" id="hexlogo" alt="rOpenSci"></a>
    <a class="navbar-brand me-2" href="../index.html">emld</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Released version">0.5.1</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"></ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/tutorial.html">emld Tutorial</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">Changelog</a>
</li>
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/ropensci/emld/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.png" class="logo" alt=""><h1>Coerce an EML file or object into an emld object.</h1>
      <small class="dont-index">Source: <a href="https://github.com/ropensci/emld/blob/HEAD/R/as_emld.R" class="external-link"><code>R/as_emld.R</code></a></small>
      <div class="d-none name"><code>as_emld.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>Coerce an EML file or object into an emld object.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">as_emld</span><span class="op">(</span><span class="va">x</span>, from <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"guess"</span>, <span class="st">"xml"</span>, <span class="st">"json"</span>, <span class="st">"list"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>x</dt>
<dd><p>path to an EML file</p></dd>


<dt>from</dt>
<dd><p>explicit type for the input format. By default, will
attempt to guess the format, but it always safer to specify the
input format. This is essential for literal text strings or raw
vectors where the type cannot be guessed by the R object class
or file extension of the input.</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p>an emld object</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span> <span class="va">hf205</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata/hf205.xml"</span>, package<span class="op">=</span><span class="st">"emld"</span><span class="op">)</span></span></span>
<span class="r-in"><span> <span class="fu">as_emld</span><span class="op">(</span><span class="va">hf205</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> access:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   authSystem: knb</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   order: allowFirst</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   scope: document</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   allow:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - principal: uid=HFR,o=lter,dc=ecoinformatics,dc=org</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     permission: all</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - principal: public</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     permission: read</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> additionalMetadata:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> - metadata:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     additionalClassifications:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       status: ongoing</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       researchTopic: community</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       studyType:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - short-term measurement</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - modeling</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> - metadata:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     additionalLinks:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       url:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - name: Ecophysiology of Carnivorous Plants Worldwide 1980-2011</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf168</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - name: Effects of Prey Availability on Sarracenia Physiology at Harvard Forest</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           2005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf109</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - name: Food Web of Sarracenia Purpurea in United States and Canada since 1999</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf193</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - name: Thresholds and Regime Shifts at Four LTER Sites (CCE, JRN, PAL, SBC)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           1951-2009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf170</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - name: Nitrogen Cycling Dynamics in Sarracenia Purpurea at Harvard Forest 2004-2005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf096</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - name: Organic and Inorganic Nitrogen Uptake by Sarracenia Purpurea at Harvard</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           Forest and Fort Albany ON 2007</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf146</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - name: Prey Capture by Carnivorous Plants Worldwide 1923-2007</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf111</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - name: Sarracenia Purpurea Prey Capture at Harvard Forest 2008</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf114</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - name: Allochthonous Nutrients in the Sarracenia Microecosystem at Harvard</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           Forest 2005-2007</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf098</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> dataset:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   id: HF205</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   alternateIdentifier: HF205</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   title: Thresholds and Tipping Points in a Sarracenia Microecosystem at Harvard Forest</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     since 2012</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   pubDate: '2012'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   abstract:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     section: '&lt;para&gt;The primary goal of this project is to determine experimentally</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       the amount of lead time required to prevent a state change. To achieve this</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       goal, we will (1) experimentally induce state changes in a natural aquatic ecosystem</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - the Sarracenia microecosystem; (2) use proteomic analysis to identify potential</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       indicators of states and state changes; and (3) test whether we can forestall</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       state changes by experimentally intervening in the system. This work uses state-of-the</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       art molecular tools to identify early warning indicators in the field of aerobic</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       to anaerobic state changes driven by nutrient enrichment in an aquatic ecosystem.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       The study tests two general hypotheses: (1) proteomic biomarkers can function</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       as reliable indicators of impending state changes and may give early warning</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       before increasing variances and statistical flickering of monitored variables;</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       and (2) well-timed intervention based on proteomic biomarkers can avert future</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       state changes in ecological systems.&lt;/para&gt;'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   intellectualRights:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     section: '&lt;para&gt;This dataset is released to the public and may be freely downloaded.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       Please keep the designated Contact person informed of any plans to use the dataset.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       Consultation or collaboration with the original investigators is strongly encouraged.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       Publications and data products that make use of the dataset must include proper</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       acknowledgement. For more information on LTER Network data access and use policies,</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       please see: http://www.lternet.edu/data/netpolicy.html.&lt;/para&gt;'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   distribution:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     online:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       url:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         function: information</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         url: http://harvardforest.fas.harvard.edu:8080/exist/xquery/data.xq?id=hf205</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   coverage:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     geographicCoverage:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       geographicDescription: Harvard Forest Greenhouse, Tom Swamp Tract (Harvard Forest)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       boundingCoordinates:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         westBoundingCoordinate: '-72.29'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         eastBoundingCoordinate: '-72.10'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         northBoundingCoordinate: '+42.55'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         southBoundingCoordinate: '+42.42'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         boundingAltitudes:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           altitudeMinimum: '160'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           altitudeMaximum: '330'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           altitudeUnits: meter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     temporalCoverage:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       rangeOfDates:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         beginDate:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           calendarDate: '2012-06-01'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         endDate:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           calendarDate: '2013-12-31'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     taxonomicCoverage:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       taxonomicClassification:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         taxonRankName: genus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         taxonRankValue: Sarracenia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         taxonomicClassification:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           taxonRankName: species</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           taxonRankValue: purpurea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   contact:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     individualName:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       givenName: Aaron</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       surName: Ellison</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     organizationName: Harvard Forest</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     address:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       deliveryPoint: 324 North Main Street</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       city: Petersham</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       administrativeArea: MA</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       postalCode: '01366'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       country: USA</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     phone:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       phonetype: voice</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       phone: (978) 724-3302</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     electronicMailAddress: aellison@fas.harvard.edu</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   publisher:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     organizationName: Harvard Forest</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     address:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       deliveryPoint: 324 North Main Street</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       city: Petersham</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       administrativeArea: MA</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       postalCode: '01366'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       country: USA</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     onlineUrl: http://harvardforest.fas.harvard.edu</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     phone:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - phonetype: voice</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       phone: (978) 724-3302</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - phonetype: fax</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       phone: (978) 724-3595</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   methods:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     methodStep:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       description:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         section:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         - '&lt;title&gt;General Protocols&lt;/title&gt;&lt;para&gt;1. Field methods. All experiments</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           will be carried out in the greenhouse at Harvard Forest. We have developed</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           an instrumentation system that allows us to collect continuous dissolved</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           [O2] measurements: dedicated micro-probes (DO-166MT; Lazar Research Laboratories:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           http://www.shelfscientific.com/) connected to multiplexers and data loggers</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           (AM16/32B multiplexer, CR-1000 datalogger and control system [Campbell Scientific:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           http://www.cambellsci.com]). The initial ecosystem composition in all experimental</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           plants will be standardized by seeding each pitcher with a 10-ml inoculum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           of liquid collected from pitchers growing at Tom Swamp.  In all experiments,</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           prey will be supplied to pitchers as standardized aliquots of dried and</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           finely ground bald-faced hornets (Dolichovespula maculata; Hymenoptera:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           Vespidae), which we collect in quantity throughout New England. Both hornets</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           and ants (the latter are the dominant prey of S. purpurea) are hymenoptera,</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           and have nearly identical C:N ratios (hornets: 3.97; common bog-dwelling</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           ants [Tapinoma sessile and Myrmica lobifrons]: 3.37), but on average hornets</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           have greater than 100 times the dry mass of these ants, and are easier to</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           collect and process as a standardized food source. Additions of prey, either</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           as large "pulses" or chronic "presses" are analogous to the enrichment and</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           eutrophication that occur in aquatic "green" food webs in which phytoplankton</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           abundance is boosted through addition of limiting nutrients. In "brown"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           food webs such as the Sarracenia microecosystem, detritus - not primary</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           production - is at the base of the web, and our treatments boost this material</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           as would happen through increases in arthropod prey capture78 or through</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           nitrogen-enriched precipitation.&lt;/para&gt;&lt;para&gt;2. Proteomic analysis. Proteomic</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           profiles of microbial communities are determined after separating the microbial</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           fraction from the pitcher fluid, prey, and other detritus. The microbial</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           "pellet" is subjected to SDS-PAGE (sodium dodecyl sulfate polyacrylamide</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           gel) electrophoresis; bands are cut out and digested in-gel with trypsin.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           Tryptic peptides are subjected to LC-MS/MS (liquid chromatography tandem</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           mass spectrometry) for peptide and protein identification. Absolute abundance</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           of peptides and proteins are quantified using AQUA (Absolute QUAntification)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           analysis109.&lt;/para&gt;'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         - '&lt;title&gt;Specific Experiments&lt;/title&gt;&lt;para&gt;Experiment #1. Effects of nutrient</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           enrichment on state changes and [O2] profiles. This experiment alters nutrient</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           enrichment rates to characterize the [O2] profile and the transition to</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           the anaerobic state. The experimental design is a one-way layout with 5</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           treatment groups: one control (no enrichment) and 4 enrichment levels (0.125,</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           0.25, 0.5, 1.0 mg prey added ml-1 d-1). One plant is assigned to each treatment</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           group, and the entire set is replicated 6 times over successive weeks. [O2]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           is monitored continuously for 4 days to characterize state changes and tipping</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           points under different enrichment rates. This experiment tracks a continuous</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           [O2] profile but does not include proteomic analysis. The purpose of Experiment</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           #1 is to identify an enrichment rate E that generates a long pre-tipping</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           period before transition time T to the anaerobic state. This enrichment</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           rate will be used in Experiments #2 - #4.&lt;/para&gt;&lt;para&gt;Experiment #2. Identification</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           of early intervention time and characterization of aerobic and anaerobic</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           proteomes. This experiment will use the single enrichment rate E determined</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           from Experiment #1 and impose different intervention times I at which nutrient</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           enrichment will be terminated. Thus, this experiment will identify the latest</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           time I* at which it is possible to intervene and stop the transition to</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           the anaerobic state by halting enrichment. The [O2] profile will again be</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           monitored continuously over 10 days to measure the state of the system.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           From Experiment #1, the transition time T to the anaerobic state with no</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           intervention will be known. We will use one control group (no prey addition)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           and ten levels of intervention time (all with the same enrichment rate E)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           as a proportion of T (0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0).</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           Six plants will be assigned randomly to each of the 11 treatments in a randomized</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           one-way layout and [O2] profiles will be monitored continuously. In addition</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           to the [O2] profiles, we will also characterize the protein profiles of</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           aerobic and anaerobic pitchers in all 11 treatment groups at the end of</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           the experiment.&lt;/para&gt;&lt;para&gt;After the plants are harvested, we will create</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           proteomic profiles of the predominantly bacterial portion (centrifuged pellet)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           of the pitcher fluid from each plant, as described in General Protocols.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           Thus, 66 separate pellet-fraction samples will be analyzed by SDS-PAGE.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           After examining the SDS-PAGE profiles, approximately ten proteins that show</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           dynamic patterns consistent with state change and five that do not change</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           will be cut from the gel, subjected to in-gel tryptic digestion and a portion</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           of the tryptic peptides will be analyzed by LC-MS/MS. Using these data,</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           we will choose three identified peptides from each protein for peptide synthesis</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           such that each synthesized peptide contains stable isotope labels (AQUA</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           peptides) to distinguish them by mass from the native peptides. We will</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           then quantify all 45 of the native peptides from the original samples using</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           a known amount of each AQUA peptide spiked into the tryptic digest. The</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           AQUA analysis of proteins that do not show changes will be used for normalization</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           between samples. These data will be used to independently identify the current</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           state of the system and forecast the time-to-transition.&lt;/para&gt;&lt;para&gt;We</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           will use Sequest searches for initial identification of peptides; relevant</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           scores including Xcorr and Delta Cn values will be given for each peptide.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           Other peptides will be identified by de novo sequencing using PepNovo; all</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           PepNovo scores will likewise be given including any N- or C-terminal gaps.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           Mass error in ppm will be reported for each precursor ion. We will use standard</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           multivariate analysis to search for distinctive proteomic profiles114 that</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           characterize aerobic and anaerobic ecosystems, as well as ecosystems that</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           developed with and without inputs of photosynthetic O2 and plant metabolites.&lt;/para&gt;&lt;para&gt;Experiment</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           #3. Identification of diagnostic proteins. Using Experiments #1 and 2, we</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           will have identified an enrichment rate E with a long pre-tipping period</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           and an intervention time I* before which mitigation and termination of enrichment</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           will prevent eutrophication. Experiment #3 will characterize the mean and</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           variance of the protein profile before and after I*. We are especially interested</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           in identifying proteins that increase rapidly in abundance (or variance)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           well before the onset of flickering in [O2] and before the transition time</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           T from the aerobic to the anaerobic state.&lt;/para&gt;&lt;para&gt;A cohort of 100 plants</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           all will be fed at rate E (determined from Experiment #1), with intervention</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           time I* determined from Experiment #2, although no intervention will be</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           used in this "press" experiment so that we can contrast proteins before</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           and after the state change. At seven times before I* and three times after</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           I*, we will harvest 10 randomly chosen plants. At each prescribed harvest</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           time, we will measure [O2] and collect samples from each plant for proteomic</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           screening using both SDS-PAGE and AQUA analysis. This experiment will identify</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           proteins that rise quickly in abundance during the pre- I* period and can</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           be used as early indicators of a future tipping point. Because different</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           plants will be harvested at each time period, this is a one-way ANOVA design,</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           with pre-and post- I* a priori contrasts. A randomization test will be used</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           to determine whether variances in protein expression differ through time.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           During these analyses we will use the data from the AQUA peptides and from</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           known amounts of protein standards, such as bovine serum albumin, to approximate</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           the amount of protein in a given coomassie-stained SDS-PAGE gel band. The</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           reason for doing this is to provide a fast "real-time" assay based just</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           on expression in the SDS-PAGE. This rapid assay will be used in Experiment</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           #4.&lt;/para&gt;&lt;para&gt;Experiment # 4. Proof-of-application. This experiment will</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           provide a benchmark test of our methods and their ability to correctly identify</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           tipping points. A cohort of 100 plants will each be fitted with [O2] probes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           and started on the enrichment regime. Two times per day, we will collect</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           3 plants each, pool their contents, and conduct a rapid screen in the lab</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           with SDS-PAGE for the diagnostic proteins that were identified in Experiment</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           #3. We will use the protein expression in the gel to delineate an "early"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           and a "late" mitigation strategy. As soon as diagnostic proteins measured</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           in the SDS-gels are at abundances that signal we are at 0.5×I* - approximately</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           one-half of the way to the latest intervention time - we will randomly select</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           one third of the remaining plants for mitigation and termination of enrichment</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           (the "early" mitigation strategy). We will continue to harvest plants from</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           the remainder of the cohort and monitor proteins. As soon as diagnostic</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           proteins signal we are at 0.75 times I*, we will randomly select one half</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           of the remaining plants for mitigation and termination of enrichment (the</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           "late" mitigation strategy). The remaining plants (approximately one sixth</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           to one third of the original cohort) will continue to be enriched. We will</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           monitor [O2] in all 3 groups (no-mitigation control, early mitigation, late</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           mitigation) until all plants reach a new [O2] equilibrium. If the protein</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           markers are successful, the proportion of food webs that remain aerobic</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           will be significantly higher in the two mitigation treatments than in the</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           no-mitigation control.&lt;/para&gt;'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   dataTable:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     id: hf205-01</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     entityName: hf205-01-TPexp1.csv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     entityDescription: tipping point experiment 1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     physical:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       id: '1354212701790'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       objectName: hf205-01-TPexp1.csv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       dataFormat:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         textFormat:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           numHeaderLines: '1'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           recordDelimiter: \r\n</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           attributeOrientation: column</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           simpleDelimited:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             fieldDelimiter: ','</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       distribution:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         online:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           url:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             function: download</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             url: http://harvardforest.fas.harvard.edu/data/p20/hf205/hf205-01-TPexp1.csv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     attributeList:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       attribute:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - id: '1354213311470'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeName: run.num</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeDefinition: 'which run number (=block). Range: 1 – 6. (integer)'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         measurementScale:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           nominal:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             nonNumericDomain:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>               textDomain:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 definition: which run number</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - id: '1354213311471'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeName: year</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeDefinition: year, 2012</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         measurementScale:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           dateTime:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             formatString: YYYY</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - id: '1354213311472'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeName: day</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeDefinition: 'Julian day. Range: 170 – 209.'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         measurementScale:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           dateTime:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             formatString: DDD</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - id: '1354213311473'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeName: hour.min</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeDefinition: hour and minute of observation. Range 1 – 2400 (integer)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         measurementScale:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           dateTime:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             formatString: hhmm</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - id: '1354213311474'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeName: i.flag</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeDefinition: is variable Real, Interpolated or Bad (character/factor)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         measurementScale:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           nominal:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             nonNumericDomain:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>               enumeratedDomain:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 codeDefinition:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: R</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: real</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: I</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: interpolated</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: B</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: bad</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - id: '1354213311475'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeName: variable</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeDefinition: what variable being measured in what treatment (character/factor).</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         measurementScale:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           nominal:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             nonNumericDomain:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>               enumeratedDomain:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 codeDefinition:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: control</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: no prey added</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: low</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: 0.125 mg prey added ml-1 d-1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: med.low</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: 0,25 mg prey added ml-1 d-1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: med.high</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: 0.5 mg prey added ml-1 d-1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: high</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: 1.0 mg prey added ml-1 d-1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: air.temp</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: air temperature measured just above all plants (1 thermocouple)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: water.temp</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: water temperature measured within each pitcher</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: par</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: photosynthetic active radiation (PAR) measured just</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                     above all plants (1 sensor)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       - id: '1354213311476'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeName: value.i</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         attributeDefinition: value of measured variable for run.num on year/day/hour.min.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         measurementScale:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           nominal:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             nonNumericDomain:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>               enumeratedDomain:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 codeDefinition:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: control</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: '% dissolved oxygen'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: low</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: '% dissolved oxygen'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: med.low</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: '% dissolved oxygen'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: med.high</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: '% dissolved oxygen'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: high</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: '% dissolved oxygen'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: air.temp</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: degrees C</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: water.temp</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: degrees C</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 - code: par</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   definition: micromoles m-1 s-1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     numberOfRecords: '9999'</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   creator:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - individualName:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       givenName: Aaron</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       surName: Ellison</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - individualName:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       givenName: Nicholas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       surName: Gotelli</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   associatedParty:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - individualName:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       givenName: Benjamin</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       surName: Baiser</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     role: Researcher</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - individualName:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       givenName: Jennifer</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       surName: Sirota</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     role: Researcher</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   keywordSet:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - keywordThesaurus: LTER controlled vocabulary</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     keyword:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - bacteria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - carnivorous plants</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - genetics</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - thresholds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - keywordThesaurus: LTER core area</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     keyword:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - populations</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - inorganic nutrients</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - disturbance</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - keywordThesaurus: HFR default</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     keyword:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - Harvard Forest</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - HFR</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - LTER</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     - USA</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   otherEntity:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - id: hf205-02</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     entityName: hf205-02-mathematica-oxygen.nb</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     entityDescription: Mathematica code for oxygen model (nb)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     physical:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       objectName: hf205-02-mathematica-oxygen.nb</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       dataFormat:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         externallyDefinedFormat:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           formatName: Mathematica notebook</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       distribution:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         online:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           url:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             function: download</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             url: http://harvardforest.fas.harvard.edu/data/p20/hf205/hf205-02-mathematica-oxygen.nb</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     entityType: Mathematica code</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   - id: hf205-03</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     entityName: hf205-03-mathematica-oxygen.pdf</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     entityDescription: Mathematica code for oxygen model (pdf)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     physical:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       objectName: hf205-03-mathematica-oxygen.pdf</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       dataFormat:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         externallyDefinedFormat:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           formatName: pdf</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       distribution:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         online:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           url:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             function: download</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             url: http://harvardforest.fas.harvard.edu/data/p20/hf205/hf205-03-mathematica-oxygen.pdf</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     entityType: Mathematica code</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> packageId: knb-lter-hfr.205.4</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> schemaLocation: eml://ecoinformatics.org/eml-2.1.0 http://nis.lternet.edu/schemas/EML/eml-2.1.0/eml.xsd</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> scope: system</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> system: hfr</span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><!-- begin footer --><div class="footer">
    <div class="container">
        <div class="row start top-4 bottom-8">
            <div class="col-2"> <img id="footerlogo" src="https://ropensci.org/img/icon_short_white.svg"></div>
            <div class="col-10">
                <div class="row">
                    <div class="col-md-4 col-xs-6">
                        <a href="https://github.com/ropensci" target="_blank" class="external-link"><div class="icon fab fa-github"></div></a>
                        <a href="https://github.com/ropenscilabs" target="_blank" class="external-link"><div class="icon fa fa-flask"></div></a>
                        <a href="https://hachyderm.io/@ropensci" target="_blank" class="external-link"><div class="icon fab fa-mastodon"></div></a>
                        <a href="https://vimeo.com/ropensci" target="_blank" class="external-link"><div class="icon fab fa-vimeo"></div></a>
                    </div>
                </div>
                <div class="row top-4">
                    <div class="col-md-2 col-sm-4">
                        <ul><h5 class="bottom-2">About</h5>
                            <li><a href="https://ropensci.org/about" class="external-link">About rOpenSci</a></li>
                            <li><a href="https://ropensci.org/software-review" class="external-link">Software Review</a></li>
                            <li><a href="https://ropensci.org/about#team" class="external-link">Our Team</a></li>
                            <li><a href="https://ropensci.org/careers" class="external-link">Jobs</a></li>
                            <li><a href="https://ropensci.org/donate" class="external-link">Donate</a></li>
                            <li><a href="https://ropensci.org/contact" class="external-link">Contact Us</a></li>
                        </ul></div>
                    <div class="col-md-3 col-sm-4">
                        <ul><h5 class="bottom-2">Community</h5>
                            <li><a href="https://ropensci.org/community/" class="external-link">Our Community</a></li>
                            <li><a href="https://ropensci.org/commcalls/" class="external-link">Community calls</a></li>
                            <li><a href="https://ropensci.org/events/" class="external-link">Events</a></li>
                            <li><a href="https://discuss.ropensci.org/" class="external-link">Join the Discussion</a></li>
                            <li><a href="https://ropensci.org/code-of-conduct" class="external-link">Code of conduct</a></li>
                        </ul></div>
                    <div class="col-md-2 col-sm-4">
                        <ul><h5 class="bottom-2">Resources</h5>
                            <li><a href="https://ropensci.org/packages/" class="external-link">Packages</a></li>
                            <li><a href="https://ropensci.org/usecases/" class="external-link">Use Cases</a></li>
                            <li><a href="https://ropensci.org/talks-papers/" class="external-link">Talks &amp; Publications</a></li>
                            <li><a href="https://docs.ropensci.org/" class="external-link">Documentation</a></li>
                            <li><a href="https://ropensci.org/news/" class="external-link">Newsletter</a></li>
                            <li><a href="https://ropensci.org/how-to-cite-ropensci/" class="external-link">Cite rOpenSci</a></li>
                        </ul></div>
                    <div class="col-md-4 col-xs-12">
                        <h5 class="bottom-2"></h5>

                        <p>rOpenSci is a fiscally sponsored project of <a href="http://numfocus.org" class="external-link">NumFOCUS</a>.</p>

                    </div>
                </div>
            </div>
        </div>
    </div>
</div>
<!-- / end footer -->


    </footer></div>

  

  

  </body></html>

